### PR TITLE
feat: Switch fabric/mod/idstore to use Couchdb

### LINKED ISSUE #34: Switch fabric/mod/idstore to use Couchdb

### PR DESCRIPTION
**Pull request by fqutishat**
Signed-off-by: Firas Qutishat <firas.qutishat@securekey.com>

closes #34

### CODE REVIEW

**Comment by fqutishat** [comment]
verify-unitest

File: mod/peer/idstore/store.go [hunk 1]
```diff
@@ -8,9 +8,10 @@ package idstore
 
 import (
 	"github.com/hyperledger/fabric/core/ledger/kvledger/idstore"
+	s "github.com/trustbloc/fabric-peer-ext/pkg/idstore"
 )
 
 // OpenIDStore open idstore
 func OpenIDStore(path string) idstore.IDStore {
-	return idstore.OpenIDStore(path)
+	return s.OpenIDStore(path)
```
Comment (troyronda) [inline mod/peer/idstore/store.go]: i assume there is a TODO to allow for configurable ID stores?

Comment (fqutishat) [reply mod/peer/idstore/store.go]: yes

**Comment by fqutishat** [comment]
verify-unitest

**Pull request merged**